packages/jsii-pacmak/test/expected.jsii-calc-lib/dotnet/Amazon.JSII.Tests.CalculatorPackageId.LibPackageId/.jsii

```diff
@@ -92,6 +92,25 @@
       ],
```

Comment:
I don't think key.snk is supposed to be checked in, is it?

Reply:
No it is not, and I definitely remember *REMOVING* it from the commit several times, and it is even in my global ignores list. WHAT THE ACTUAL FUNK, GIT?